seed/challenges/02-data-visualization-certification/data-visualization-certificate.json

```diff
@@ -118,7 +118,34 @@
           ""
         ]
       ],
-      "titleEs": "Reclama tu certificado de Visualización de datos"
+      "titleEs": "Reclama tu certificado de Visualización de datos",
```

Comment:
@elazzabi ah, yes, you're right, it was false positive. It is so unusual that the title comes after description, that's why I was confused, in full view everything is OK. It is better to keep the title before the description though, like in other files.